internal/kaniko/types.go

```diff
@@ -14,6 +14,8 @@ type Config struct {
 	Destination string `json:"destination,omitempty"`
 	// RegistryMirrors contains registries used to pull images.
 	RegistryMirrors string `json:"registryMirrors,omitempty"`
+	// SendEvent sets whether to save the metadata for the published container images in the CloudBees platform.
+	SendEvent bool `json:"sendEvent,omitempty"`
```

Comment:
@kushicp based on discussion with Mariluz, I've removed the flag as an input parameter:https://docs.google.com/document/d/1I8BUx0XpkWMiiHDz8WgDtcza8DaeEgOvqqpE94JFa9s/edit?disco=AAABYDaDUFo&pli=1. We will always send the artifact info from this action for the artifacts it published.

Reply:
Yes, I have removed it too. Weird that I don't see it in my local repo, but see it in this repo. Will recommit the file and this should reflect